arm-web/2016-08-01/WebApps.json

```diff
@@ -1736,6 +1736,140 @@
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the app.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/SiteConfigurationSnapshotInfo"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web/snapshots/{snapshotId}": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets a snapshot of the configuration of an app, such as platform version and bitness, default documents, virtual applications, Always On, etc.",
+        "description": "Gets a snapshot of the configuration of an app, such as platform version and bitness, default documents, virtual applications, Always On, etc.",
+        "operationId": "WebApps_GetConfigurationSnapshot",
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the app.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "snapshotId",
+            "in": "path",
+            "description": "The ID of the snapshot to read.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SiteConfig"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web/snapshots/{snapshotId}/recover": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets a snapshot of the configuration of an app, such as platform version and bitness, default documents, virtual applications, Always On, etc.",
```

Comment:
I'll change this one to "Reverts the configuration of an app to a previous snapshot"